components/footer/Footer.vue

```diff
@@ -69,6 +77,8 @@ export default {
           const amount = await EthHelper.queryEthBalance(wallet);
           opt.ETH = Number(new BigNumber(amount).dividedBy(ONE_LIKE).toFixed(4));
         }
+        opt.isEmailVerified = !!isEmailVerified;
+        opt.isAvatarUpdated = this.AVATAR_DEFAULT_PATH !== avatar;
```

Comment:
this boolean is more likely `hasAvatar`

Reply:
This part is moved to cloud function, so it is updated at firebase repo.